python/lsst/ctrl/bps/restart.py

```diff
@@ -53,4 +56,8 @@ def restart(wms_service, run_id):
         service = wms_service_class({})
     else:
         service = wms_service
-    return service.restart(run_id)
+    args = [run_id]
```

Comment:
Could you add a comment that this is just a temporary solution to unblock passing `compute_site` to Parsl plugin without breaking other plugins?

Reply:
Thanks for this feedback Mikolaj, and thank you in particular for your help getting this set up yesterday. I take your comment above that this is indeed a temporary solution. With that in mind, and considering that a) this isn't a feature that I urgently need right now, and b) I've had trouble testing this feature using the BPS/parsl plugin (where `bps restart` is having issues running successfully), then I'm going to go ahead and suggest that this new `computeSite` argument also be removed from `bps restart` as well.

Reply:
With that in mind, following this PR, `bps submit` would be the only remaining BPS subcommand that would include the new `computeSite` argument.